e2etest/bgptests/bgp.go

```diff
@@ -1235,6 +1235,9 @@ var _ = ginkgo.Describe("BGP", func() {
 			err := ConfigUpdater.Update(resources)
 			framework.ExpectNoError(err)
 
+			// Sleep to let the speaker time to reload the config and produce logs
+			time.Sleep(5 * time.Second)
```

Comment:
how does this solve something when the eventually afterwards runs for 2 minutes?

Reply:
I thought that the logs are taken from the pod we fetch outside of the eventually block,
but in second look you're right and it just uses the pod name and fetches the latest logs.
Closing this one for now.